src/plugins/intel_cpu/src/emitters/snippets/jit_snippets_call_args.hpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <cstdint>
+#include <cstddef>
+#include "dnnl_types.h"
+
+namespace ov {
+namespace intel_cpu {
+
+#define SNIPPETS_MAX_SNIPPETS_DIMS 12
```

Comment:
I think this value should be equal to 11 on x64 and 23 on ARM because this value means size of static src and dst arrays in `jit_snippets_call_args` on L34-35.
Also I think we should rename it smth like `SNIPPETS_MAX_DATA_POINTERS` 
Could you please set the different values for different archs?
Thanks!

Reply:
Applied. Thanks Alexandra!